sefaria/model/topic.py

```diff
@@ -44,7 +44,8 @@ class Topic(abst.SluggedAbstractMongoRecord, AbstractTitledObject):
         'good_to_promote',
         'description_published',  # bool to keep track of which descriptions we've vetted
         'isAmbiguous',  # True if topic primary title can refer to multiple other topics
-        "data_source"  #any topic edited manually should display automatically in the TOC and this flag ensures this
+        "data_source",  #any topic edited manually should display automatically in the TOC and this flag ensures this
+        'image'
```

Comment:
We are starting use of a tool to validate attributes that are dictionaries and need to have their structure validated. Would be good to use here as well.

Reply:
Great, pushed! Let me know what you think :)